tests/tests_configure_ha_cluster.yml

```diff
@@ -0,0 +1,55 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Ensure that the role configures HA cluster
+  hosts: all
+  vars:
+    mssql_accept_microsoft_odbc_driver_17_for_sql_server_eula: true
+    mssql_accept_microsoft_cli_utilities_for_sql_server_eula: true
+    mssql_accept_microsoft_sql_server_standard_eula: true
+    mssql_password: "p@55w0rD"
+    mssql_edition: Evaluation
+    mssql_ha_db_name: ExampleDB
+    mssql_ha_listener_port: 5022
+
+  tasks:
+    - name: Open required firewall ports
+      vars:
+        firewall:
+          - port: "{{ mssql_ha_listener_port }}/tcp"
+            zone: public
+            state: enabled
+            permanent: true
+            runtime: true
+      include_role:
+        name: fedora.linux_system_roles.firewall
```

Comment:
hmm - I guess this is ok for now - for CI you'll need to add a meta/requirements.yml with the `fedora.linux_system_roles` collection 
```
collections:
  - name: fedora.linux_system_roles
```
tox -e qemu[-ansible-core-x.xx] will also use this file to install necessary collections

Reply:
I'm planing to take a look at the CI and how to implement it a bit later